code/modules/mob/living/simple_animal/distortion/myth/another_day_work.dm

```diff
@@ -38,9 +38,17 @@
 	gender = MALE
 	//The Egoist's outfit, which should usually be civilian unless you want them to be a fixer or something.
 	egoist_outfit = /datum/outfit/job/civilian
+	//The Egoist's starting stats, all across the board. MUST BE AT LEAST THE MINIMUM TO EQUIP THE EGO_GEAR IF ANY
+	egoist_attributes = 40
 	//Loot on death; distortions should be valuable targets in general.
 	loot = list(/obj/item/documents/ncorporation)
 
+//Set random stats for the egoist
+/mob/living/simple_animal/hostile/distortion/another_day_work/Initialize()
+    . = ..()
+    egoist_attributes = min(rand(0, 130), rand(0, 130))
```

Comment:
Change the 0 to egoist_attributes

Reply:
Alright, I changed it. Should be alright like that I guess.